crates/core/tedge_agent/src/agent.rs

```diff
@@ -1,36 +1,35 @@
 use crate::{
     error::AgentError,
-    http_rest,
     restart_operation_handler::restart_operation,
     state::{
         AgentStateRepository, RestartOperationStatus, SoftwareOperationVariants, State,
         StateRepository, StateStatus,
     },
 };
-use agent_interface::{
+use flockfile::{check_another_instance_is_not_running, Flockfile};
+use tedge_api::{
     control_filter_topic, software_filter_topic, Jsonify, OperationStatus, RestartOperationRequest,
     RestartOperationResponse, SoftwareError, SoftwareListRequest, SoftwareListResponse,
     SoftwareRequestResponse, SoftwareType, SoftwareUpdateRequest, SoftwareUpdateResponse,
 };
-use flockfile::{check_another_instance_is_not_running, Flockfile};
 
 use mqtt_channel::{Connection, Message, PubChannel, StreamExt, SubChannel, Topic, TopicFilter};
 use plugin_sm::{
     operation_logs::{LogKind, OperationLogs},
     plugin_manager::{ExternalPlugins, Plugins},
 };
 
-use crate::http_rest::HttpConfig;
 use std::process::Command;
 use std::{convert::TryInto, fmt::Debug, path::PathBuf, sync::Arc};
+use tedge_api::health::{health_check_topics, send_health_status};
+use tedge_api::http_rest::HttpConfig;
```

Comment:
Minor. `tedge_api` is used in 3 different places in `agent.rs`. It would be better to have all these grouped.